spec/services/idv/socure_user_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Idv::SocureUser do
+  around do |ex|
+    REDIS_THROTTLE_POOL.with { |client| client.flushdb }
+    ex.run
+    REDIS_THROTTLE_POOL.with { |client| client.flushdb }
+  end
+
```

Comment:
```suggestion
```

I think this shouldn't be necessary since it should already be cleared before every test: https://github.com/18F/identity-idp/blob/73795c9eeada8b109269001de807fd397185a22d/spec/rails_helper.rb#L114